cmd/rigs/cmd/publish_data.go

```diff
@@ -218,3 +180,35 @@ func attrsJobs(ctx context.Context, s local.Store, jobID *int) ([]wpool.Job, err
 	}
 	return jobs, nil
 }
+
+func lookupsJob(s local.Store, jobID *int) wpool.Job {
+	return wpool.Job{
+		ID: wpool.JobID(*jobID),
+		ExecFn: func(ctx context.Context) (interface{}, error) {
+			rendersCid, err := s.Cid(ctx, "renders")
+			if err != nil {
+				return nil, fmt.Errorf("querying images cid: %v", err)
+			}
+			layersCid, err := s.Cid(ctx, "layers")
+			if err != nil {
+				return nil, fmt.Errorf("querying layers cid: %v", err)
+			}
+			lookups := tableland.Lookups{
+				RendersCid:           rendersCid,
+				LayersCid:            layersCid,
+				ImageFullName:        "image_full.png",
+				ImageFullAlphaName:   "image_full_alpha.png",
+				ImageMediumName:      "image_medium.png",
+				ImageMediumAlphaName: "image_medium_alpha.png",
+				ImageThumbName:       "image_thumb.png",
+				ImageThumbAlphaName:  "image_thumb_alpha.png",
+				AnimationBaseURL:     "https://tableland.xyz/rigs/animate?id=", // TODO: Update when we know it.
```

Comment:
This needs to be updated once we know the url of the animation app cc @joewagner